### ISSUE TITLE
Users are unable to access timesheets.

### ISSUE THREAD

**Issue by adunkman** [18F/tock#1509]
Reported [via Slack](https://gsa-tts.slack.com/archives/C1JFYCX3P/p1672928180823359), [related log entries](https://logs.fr.cloud.gov/app/dashboards#/view/App-Errors?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2023-01-05T13:00:00.000Z',to:'2023-01-05T16:00:00.000Z'))&_a=(description:'',filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,key:query,negate:!f,type:query_string,value:'*'),query:(query_string:(analyze_wildcard:!t,query:'*')))),fullScreenMode:!f,options:(darkTheme:!f),query:(language:kuery,query:''),timeRestore:!f,title:'App%20-%20Errors',viewMode:view)) (select row > view surrounding documents to see error traces).

Apparent source error: 

```
django.db.utils.IntegrityError: null value in column "total_allocation_hours" violates not-null constraint
```

**Comment by carjug** [comment]
Solved by #1510, deploying imminently.

**Comment by adunkman** [comment]
Thank you for both digging in and understanding the issue (fixing prod can be stressful!) and taking on the task of getting weekly billing utilization reported correctly in the first place! 💖 💖 💖
